mbzirc_custom/mbzirc_naive_spinning_radar/src/MaritimeRadar.cc

```diff
@@ -0,0 +1,267 @@
+#include "MaritimeRadar.hh"
+
+#include <ignition/gazebo/Model.hh>
+#include <ignition/gazebo/Util.hh>
+#include <ignition/gazebo/components/JointPosition.hh>
+#include <ignition/plugin/Register.hh>
+
+using namespace mbzirc;
+using namespace ignition;
+using namespace gazebo;
+
+///////////////////////////////////////////////////
+MaritimeRadar::MaritimeRadar()
+{
+
+}
+
+///////////////////////////////////////////////////
+MaritimeRadar::~MaritimeRadar()
+{
+
+}
+
+///////////////////////////////////////////////////
+void MaritimeRadar::Configure(const ignition::gazebo::Entity &_entity,
+                            const std::shared_ptr<const sdf::Element> &_sdf,
+                            ignition::gazebo::EntityComponentManager &_ecm,
+                            ignition::gazebo::EventManager &_eventMgr)
+{
+  gazebo::Model model(_entity);
+
+  // TODO(arjo): This is a hack to get the laser
+  std::string laserTopic =
+    "/world/coast/model/usv/model/sensor_0/link/sensor_link/sensor/lidar/scan";
```

Comment:
we should be able to get the world name through _ecm, something like

```
auto worldEntity = _ecm.EntityByComponents(components::World());
std::string name = _ecm.Component<components::Name>(worldEntity)->Data();
```

Reply:
https://github.com/osrf/mbzirc/pull/127/commits/52aa818a3355b82bd7b16762183248f8c2b3e5c8